privileges-grantor/README.md

```diff
@@ -44,6 +42,54 @@ module "apiary-privileges-grantor" {
 
 ```
 
+The apiary-privileges-grantor lambda can be found in the public [maven repository](https://mvnrepository.com/artifact/com.expediagroup.apiary/apiary-privileges-grantor-lambda).
+The jars can be downloaded from the link provided above and uploaded to S3 via terraform as follows:
+
+```
+variable "pg_lambda_version" {
```

Comment:
OK, so the version is still available as a variable? We should still document it in the README shouldn't we?

Reply:
It is not. This is an example code that is optional and helps with fetching the lambda artefact from maven and uploads it to S3.

Reply:
OK, as long as @patduin is OK with this then so am I. @patduin ?

Reply:
see my comments on the nullResource thing. Not ok :)